src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -78,6 +78,7 @@ def load_arguments(self, _):
     with self.argument_context('backup vault backup-properties set') as c:
         c.argument('backup_storage_redundancy', arg_type=get_enum_type(['GeoRedundant', 'LocallyRedundant']), help='Sets backup storage properties for a Recovery Services vault.')
         c.argument('soft_delete_feature_state', arg_type=get_enum_type(['Enable', 'Disable']), help='Set soft-delete feature state for a Recovery Services Vault.')
+        c.argument('cross_region_restore_flag', arg_type=get_enum_type(["True", "False"]), help='Set cross-region-restore feature state for a Recovery Services Vault. Default: False.')
```

Comment:
`cross_region_restore_flag` will be treated as a `str`, you may need to convert it to bool before you pass it to SDK.

Reply:
Did the changes. Please review once.